### ISSUE TITLE
env: node\r: No such file or directory

### ISSUE THREAD

**Issue by leleliu008** [android-js/androidjs-builder#35]
run `androidjs` command on macOS occurred error:`env: node\r: No such file or directory`

**Comment by Chhekur** [comment]
> run `androidjs` command on macOS occurred error:`env: node\r: No such file or directory`

I guess, you used `yarn` to install `Android JS`, can you use `npm` instead ?

Feel free to ask in case of any problem ; )

**Comment by leleliu008** [comment]
@Chhekur Yes, I used `yarn` to install `AndroidJS Builder`. I resolved this problem by following steps:
```
vim $(command -v androidjs)
:set ff=unix
:wq
```
